collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -713,7 +687,7 @@ public void formListDownloadingComplete(HashMap<String, FormDetails> result) {
             }
             selectSupersededForms();
             mFormListAdapter.notifyDataSetChanged();
-            mDownloadButton.setEnabled(!(selectedItemCount() == 0));
+            mDownloadButton.setEnabled(!(ListViewUtils.selectedItemCount(getListView()) == 0));
```

Comment:
Same as earlier, makes more sense to use > 0 or even to create something like a hasSelectedItems(lv) predicate.